detect_secrets/audit/common.py

```diff
@@ -95,6 +95,43 @@ def get_raw_secret_from_file(
     raise SecretNotFoundOnSpecifiedLineError(secret.line_number)
 
 
+def get_all_secrets_from_file(
```

Comment:
This method is very similar to `get_raw_secret_from_file` with the difference that it search all the ocurrences of one secret in one file with one plugin. It allows to offer a very complete report of all secrets found with detect-secrets. The method `get_all_secrets_from_file` has been added to the `detect_secrets/audit/common.py` file, so it can be reused by other features in the future.